pkg/packagekit/updates.jsx

```diff
@@ -509,6 +514,116 @@ const AskRestart = ({ onIgnore, onRestart, history }) => <>
     </div>
 </>;
 
+const StatusCard = ({ updates, highestSeverity, timeSinceRefresh }) => {
+    const numUpdates = Object.keys(updates).length;
+    const numSecurity = count_security_updates(updates);
+    let stateStr;
+
+    if (numUpdates == 0)
+        stateStr = _("System is up to date");
+    else if (numUpdates == numSecurity)
+        stateStr = cockpit.ngettext("$1 security fix", "$1 security fixes", numSecurity);
+    else {
+        stateStr = cockpit.ngettext("$0 update", "$0 updates", numUpdates);
+        if (numSecurity > 0)
+            stateStr += cockpit.ngettext(", including $1 security fix", ", including $1 security fixes", numSecurity);
+    }
+    stateStr = cockpit.format(stateStr, numUpdates, numSecurity);
+
+    if (!stateStr)
+        return null;
+
+    let lastChecked;
+    if (timeSinceRefresh !== null)
+        lastChecked = cockpit.format(_("Last checked: $0"), moment(moment().valueOf() - timeSinceRefresh * 1000).fromNow());
+
+    const icon = highestSeverity ? <span className={PK.getSeverityIcon(highestSeverity)} /> : <CheckIcon color="green" />;
+
+    return (<Flex>
+        <Flex>
+            <FlexItem>{icon}</FlexItem>
+        </Flex>
+        <Flex flex={{ default: 'flex_1' }}>
+            <FlexItem>
+                <Text id="state" component={TextVariants.p}>{stateStr}</Text>
+                <Text id="last-checked" component={TextVariants.small}>{lastChecked}</Text>
+            </FlexItem>
+        </Flex>
+    </Flex>);
+};
+
+class CardsPage extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            autoUpdatesEnabled: undefined,
+            autoUpdatesType: undefined,
+            autoUpdatesDay: undefined,
+            autoUpdatesTime: undefined,
+        };
+    }
+
+    render() {
+        const cardContents = [
+            {
+                id: "status",
+                title: _("Status"),
+                className: "pk-card-info",
+                actions: (<Tooltip content={_("Check for updates")}>
+                    <Button variant="secondary" onClick={this.props.handleRefresh}><i className="fa fa-refresh" /></Button>
```

Comment:
Can you please use PF4 icons here?
It will make the migration process easier.

Reply:
I could not find a same icon in PF4, But I found spinner2 (https://pf4.patternfly.org/icons/) `<i class="pf-icon pf-icon-spinner2"></i>`, Maybe that icon is okay too

Reply:
So I used RedoIcon in the end. Fixed